OmniKit/MessageTransport/MessageBlocks/PodInfoFaultEvent.swift

```diff
@@ -63,14 +64,14 @@ public struct PodInfoFaultEvent : PodInfo, Equatable {
         if reservoirValue <= StatusResponse.maximumReservoirReading {
             self.reservoirLevel = reservoirValue
         } else {
-            self.reservoirLevel = nil
+            self.reservoirLevel =  nil
         }
         
         self.timeActive = TimeInterval(minutes: Double(encodedData[13...14].toBigEndian(UInt16.self)))
         
-        self.previousStatus = FaultEventCode(rawValue: encodedData[15])
+        self.unacknowledgedAlerts = encodedData[15]
```

Comment:
Does this field map to AlertType?

Reply:
@itsmojo This is the TT field.  Should it map to these types?https://github.com/ps2/rileylink_ios/blob/f0191d2805d782a920f16b816d20c69f9c2e8b07/OmniKit/MessageTransport/MessageBlocks/ConfigureAlertsCommand.swift#L34-L42

Reply:
In the $19 message, these are not a bitmask, but single values.  The wiki indicates it's a bitmask, so are these values translated into bit positions and used as a bitmask, or is this field the same as the `aaaaaaaa` bits in the $1d response? https://github.com/ps2/rileylink_ios/blob/omnipod-testing/OmniKit/Model/PodAlarm.swift#L11-L19